### PR TITLE
Mynewt: fix GAP/SEC/SEM tests

### PR DESCRIPTION
**Pull request by KKopyscinski**
Adjusted WIDs and test procedure to GATT Client.

### CODE REVIEW

File: autopts/ptsprojects/mynewt/gap_wid.py [hunk 1]
```diff
@@ -76,12 +72,33 @@ def hdl_wid_204(_: WIDParams):
...
+def hdl_wid_227(params: WIDParams):
+    bd_addr = btp.pts_addr_get()
+    bd_addr_type = btp.pts_addr_type_get()
+    stack = get_stack()
+
+    handle = btp.parse_handle_description(params.description)
+    if not handle:
+        return False
+
+    btp.gatt_cl_write(bd_addr_type, bd_addr, handle, '02', 1)
```
Comment (sjanc) [inline autopts/ptsprojects/mynewt/gap_wid.py:84]: maybe handle this in generic wid?

if (gatt_cl_supported)
   gatt_cl...
else
  gattc....

File: autopts/ptsprojects/mynewt/gap_wid.py [hunk 2]
```diff
@@ -76,12 +72,33 @@ def hdl_wid_204(_: WIDParams):
...
+    if (btp.verify_att_error("authentication error")):
+        btp.gap_pair()
+
+    if (btp.verify_att_error("insufficient encryption")):
+        btp.gap_pair()
+
+    return True
+
+
 def hdl_wid_242(_: WIDParams):
```
Comment (sjanc) [inline autopts/ptsprojects/mynewt/gap_wid.py:96]: hmm why there is no generic wid handler for this wid?

File: autopts/ptsprojects/mynewt/gap_wid.py (hunk 1 above)
Comment (KKopyscinski) [reply autopts/ptsprojects/mynewt/gap_wid.py:84]: hmm I think I missed this, general wid works ok here. I'll remove this custom one

File: autopts/ptsprojects/mynewt/gap_wid.py (hunk 2 above)
Comment (KKopyscinski) [reply autopts/ptsprojects/mynewt/gap_wid.py:96]: I'll move this to general, I think that maybe some IXIT value makes Zephyr path different?

**Comment by mkasenberg** [comment]
#AutoPTS run mynewt GAP/SEC/SEM

**Comment by codecoup-tester** [comment]
AutoPTS Bot results:
<details><summary>Failed tests</summary>GAP     GAP/SEC/SEM/BI-22-C             BTP ERROR<br>GAP     GAP/SEC/SEM/BI-23-C             BTP ERROR<br>GAP     GAP/SEC/SEM/BV-45-C             FAIL<br>GAP     GAP/SEC/SEM/BV-65-C             INCONC<br>GAP     GAP/SEC/SEM/BV-66-C             INCONC<br>GAP     GAP/SEC/SEM/BV-67-C             INCONC</details>
<details><summary>Successful tests</summary>GAP     GAP/SEC/SEM/BI-09-C             PASS<br>GAP     GAP/SEC/SEM/BI-10-C             PASS<br>GAP     GAP/SEC/SEM/BI-20-C             PASS<br>GAP     GAP/SEC/SEM/BI-21-C             PASS<br>GAP     GAP/SEC/SEM/BV-21-C             PASS<br>GAP     GAP/SEC/SEM/BV-22-C             PASS<br>GAP     GAP/SEC/SEM/BV-23-C             PASS<br>GAP     GAP/SEC/SEM/BV-24-C             PASS<br>GAP     GAP/SEC/SEM/BV-26-C             PASS<br>GAP     GAP/SEC/SEM/BV-27-C             PASS<br>GAP     GAP/SEC/SEM/BV-28-C             PASS<br>GAP     GAP/SEC/SEM/BV-37-C             PASS<br>GAP     GAP/SEC/SEM/BV-38-C             PASS<br>GAP     GAP/SEC/SEM/BV-39-C             PASS<br>GAP     GAP/SEC/SEM/BV-40-C             PASS<br>GAP     GAP/SEC/SEM/BV-41-C             PASS<br>GAP     GAP/SEC/SEM/BV-42-C             PASS<br>GAP     GAP/SEC/SEM/BV-43-C             PASS<br>GAP     GAP/SEC/SEM/BV-44-C             PASS<br>GAP     GAP/SEC/SEM/BV-56-C             PASS<br>GAP     GAP/SEC/SEM/BV-57-C             PASS<br>GAP     GAP/SEC/SEM/BV-58-C             PASS<br>GAP     GAP/SEC/SEM/BV-59-C             PASS<br>GAP     GAP/SEC/SEM/BV-60-C             PASS<br>GAP     GAP/SEC/SEM/BV-61-C             PASS<br>GAP     GAP/SEC/SEM/BV-62-C             PASS<br>GAP     GAP/SEC/SEM/BV-63-C             PASS<br>GAP     GAP/SEC/SEM/BV-64-C             PASS</details>

**Comment by mkasenberg** [comment]
#AutoPTS run zephyr GAP/SEC/SEM

**Comment by codecoup-tester** [comment]
Scheduled PR #https://github.com/auto-pts/auto-pts/pull/983#issuecomment-1700573940 after 10:18:42.

**Comment by codecoup-tester** [comment]
AutoPTS Bot results:
No failed test found.
<details><summary>Successful tests</summary>GAP     GAP/SEC/SEM/BI-09-C             PASS<br>GAP     GAP/SEC/SEM/BI-10-C             PASS<br>GAP     GAP/SEC/SEM/BI-20-C             PASS<br>GAP     GAP/SEC/SEM/BI-21-C             PASS<br>GAP     GAP/SEC/SEM/BI-22-C             PASS<br>GAP     GAP/SEC/SEM/BI-23-C             PASS<br>GAP     GAP/SEC/SEM/BV-21-C             PASS<br>GAP     GAP/SEC/SEM/BV-22-C             PASS<br>GAP     GAP/SEC/SEM/BV-23-C             PASS<br>GAP     GAP/SEC/SEM/BV-24-C             PASS<br>GAP     GAP/SEC/SEM/BV-26-C             PASS<br>GAP     GAP/SEC/SEM/BV-27-C             PASS<br>GAP     GAP/SEC/SEM/BV-28-C             PASS<br>GAP     GAP/SEC/SEM/BV-29-C             PASS<br>GAP     GAP/SEC/SEM/BV-37-C             PASS<br>GAP     GAP/SEC/SEM/BV-38-C             PASS<br>GAP     GAP/SEC/SEM/BV-39-C             PASS<br>GAP     GAP/SEC/SEM/BV-40-C             PASS<br>GAP     GAP/SEC/SEM/BV-41-C             PASS<br>GAP     GAP/SEC/SEM/BV-42-C             PASS<br>GAP     GAP/SEC/SEM/BV-43-C             PASS<br>GAP     GAP/SEC/SEM/BV-44-C             PASS<br>GAP     GAP/SEC/SEM/BV-45-C             PASS</details>

**Review by mkasenberg — APPROVED** [review]

**Pull request merged**